tests/fsharp/Compiler/Language/StructActivePatternTests.fs

```diff
@@ -179,11 +179,8 @@ let (|Foo|_|) x = ValueNone
 """
             [|(FSharpDiagnosticSeverity.Error, 842, (2, 3, 2, 9),
                "This attribute is not valid for use on this language element");
-              (FSharpDiagnosticSeverity.Error, 1, (2, 1, 3, 16),
-               "This expression was expected to have type
-    ''a option'    
-but here has type
-    ''b voption'    ")|]
+              (FSharpDiagnosticSeverity.Error, 3350, (2, 1, 3, 16),
```

Comment:
Feels wrong to see 'bool representation' feature mentioned here, the supplied user code has nothing to do with bools.. ?

Reply:
This feature will also allowing omitting the use of `[<return: Struct>]` on `voption`-return partial active pattern...

Reply:
The feature was renamed to "Boolean-returning and return-type-directed partial active patterns", this might be a more proper name for it.